VersionControlUI/src/DiffManager.cpp

```diff
@@ -260,7 +261,6 @@ bool DiffManager::satisfiesTargetNodeIdConstraint(Model::Node* oldNode, Model::N
 }
 
 bool DiffManager::satisfiesNameChangeVisualizationConstraints(Model::NodeIdType id)
-__attribute__((optnone))
```

Comment:
Please, be more careful when committing and avoid committing things like this. In the ideal case the commit history should contain small, but still meaningful commits. Even if you accidentally commit some debug code, you should still see if there is a good way to fix it before pushing. In this particular case you could have avoided the extra commit, by simply amending the previous one.